<launch>
  <include file="$(find pr2_machine)/$(env ROBOT).machine" />	
  <node pkg="pcl_cloud_tools" type="extract_clusters_on_table" name="extract_clusters" output="screen" respawn="true" args="free_view" machine="c2">
    <!-- Parameters --> 
    <param name="cluster_min_height" value="0.02"/>
    <param name="cluster_max_height" value="0.3"/>	
    <param name="object_cluster_tolerance" value="0.05"/>
    <param name="object_cluster_min_size" value="50"/>
    <param name="nr_cluster" value="1"/>
    <param name="object_cluster_tolerance" value="0.05"/>
    <param name="downsample" value="true"/>
    <param name="voxel_size" value="0.02"/>
    <param name="save_to_files" value="false"/>
    <param name="base_link_head_tilt_link_angle" value="0.90"/>
    <!-- <param name="point_cloud_topic" value="/camera/rgb/points_throttle"/> -->
    <param name="point_cloud_topic" value="/narrow_stereo_textured/points2"/>
    <param name="publish_token" value="false"/>	
    <!-- <param name="padding" value="0.1"/>		 -->
    <param name="z_max_limit" value="1.2"/>
 </node>
</launch>
